src/services/templates/SendOtpEmail.js

```diff
@@ -63,7 +63,7 @@ export const sendOtpEmail = (emailData) => {
               <p>To finish your registration, please use this OTP:</p>
           </div>
           <div class="button-container" style="text-align: center;">
-              <a href="#" class="button"><b>${link}</b></a>
+              <p href="#" class="button"><b>${otp}</b></p>
```

Comment:
The link href should be a link not an # and the contents of the button should read verify

`<a href=`${link}>Verify</a>

Reply:
Good morning @Kaleshe, I don't think this was supposed to be a link. It just displays a series of numbers that the user will copy and paste on the email verification page.

Reply:
![image](https://github.com/content-is-queen/creator-platform-be/assets/52130946/a680079b-8e5f-44f1-8667-5c3e6d6ece7a)

Reply:
It should be a link. So that the user can click and have the fields automatically filled in. That's how I want the feature to be on the frontend. So if you could just add in those bits here, I'll handle it on the frontend

Reply:
How about changing from OTP to a link so that when users click on the link, their accounts will be verified automatically

Reply:
Oh if that's possible, then let's do that